src/main/java/org/jabref/gui/maintable/MainTableColumnModel.java

```diff
@@ -35,7 +35,7 @@ public class MainTableColumnModel {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MainTableColumnModel.class);
     public enum Type {
-        MATCH_CATEGORY("match_category", Localization.lang("Match category")),
+        MATCH_CATEGORY("match_category"),
```

Comment:
Please add JavaDoc comment

    // Not localized, because this column is always hidden